docs/manual/scala/guide/cluster/Serialization.md

```diff
@@ -0,0 +1,161 @@
+# Serialization
+
+Out of the box, Lagom will use JSON for request and response message format for the external API of the service, using Play JSON to serialize and deserialize messages. The messages that are sent within the cluster of the service must also be serializable and so must the events that are stored by [[Persistent Entities|PersistentEntity]]. We recommend JSON for these as well and Lagom makes it easy to add Play JSON serialization support to such classes.
+
+Do not depend on Java serialization for production deployments. It is inefficient both in serialization size and speed. It is very difficult to evolve the classes when using Java serialization, which is especially important for the persistent state and events, since you must be able to deserialize old objects that were stored.
+
+Runtime overhead is avoided by not basing the serialization on reflection. Transformations to and from JSON are defined either manually or by using a built in macro - essentially doing what reflection would do, but at compile time instead of during runtime. This comes with one caveat, each top level class that can be serialized needs an explicit serializer defined. To use the Play JSON support in Lagom you need to provide such serializers for each type.
+
+The Play JSON abstraction for serializing and deserializing a class into JSON is the [Format](https://www.playframework.com/documentation/2.5.x/api/scala/index.html#play.api.libs.json.Format) which in turn is a combination of [Reads](https://www.playframework.com/documentation/2.5.x/api/scala/index.html#play.api.libs.json.Reads) and [Writes](https://www.playframework.com/documentation/2.5.x/api/scala/index.html#play.api.libs.json.Writes). The library parses JSON into a JSON tree model, which is what the `Format`s work with.
+
+
+## Enabling JSON Serialization
+
+To enable JSON Serialization there are 3 steps you need to follow.
+
+The first step is to mark your class for JSON serialization by making it implement the [Jsonable](api/index.html?com/lightbend/lagom/serialization/Jsonable.html) marker trait. 
```

Comment:
update the api links in the same way I did in the other PR, i.e. ? -> # and remove the trailing .html